arenadata/docker-compose.yaml

```diff
@@ -0,0 +1,64 @@
+---
+
+version: "3"
+services:
+  mdw:
+    image: "hub.adsw.io/library/gpdb_regress:${BRANCH_NAME}"
```

Comment:
image name has been renamed to gpdb6_regress since adb-5.x tests were implemented. Please check it.